src/utils/graphql/queries/getApplication.query.ts

```diff
@@ -21,13 +21,26 @@ export default gql`
             ...TemplateStage
           }
         }
+        # Count PREVIEW actions so we know whether to show the option to Preview
+        templateActions(condition: { trigger: ON_PREVIEW }) {
+          totalCount
+        }
       }
       user {
         ...User
       }
       org {
         ...Organisation
       }
+      # Get scheduled events so we can extend deadlines
+      triggerSchedules {
```

Comment:
Here also I would make an alias: `applicationDeadlineSchedules: triggerSchedules (.. query for eventCode)`

Reply:
I'm reluctant to alias it here, as this is actually fetching *all* trigger schedules for this application, not just the deadline ones, so calling it `applicationDeadlineSchedules` would be misleading. Wouldn't it be better to change the name when we insert a filtered version into the structure further down?

Reply:
oki, I don't think I communicated it well, was suggesting we also filter `triggerSchedules` by the eventType. But i guess it's ok to just filter it out in structure building

Reply:
Okay, got ya. I think my commit mentioned in the [comment below](https://github.com/openmsupply/conforma-web-app/pull/1239#discussion_r935540774) should take care of it. No filtering and renaming here in the query (I left it like that cos we may want to use other events at some point), but I've filtered, renamed and structured it as it goes into the full structure, so the "Overview" page doesn't need to do any additional logic.